teachertool/src/transforms/readdCriteriaToChecklist.ts

```diff
@@ -0,0 +1,28 @@
+import { stateAndDispatch } from "../state";
+import { logDebug } from "../services/loggingService";
+import { setChecklist } from "./setChecklist";
+import { Ticks } from "../constants";
+import { getCriteriaInstanceWithId } from "../state/helpers";
+
+export function readdCriteriaToChecklist(criteriaInstanceId: string) {
```

Comment:
nit: I saw this as a typo of "read" instead of "re-add". Perhaps "reAdd" or "undelete" would be clearer?